package.json

```diff
@@ -104,7 +104,9 @@
     "overrides": {
       "@types/react": "19.0.1",
       "@types/react-dom": "19.0.2",
-      "react-is": "19.0.0"
+      "react-is": "19.0.0",
+      "@renegade-fi/core": "0.4.12",
+      "viem": "$viem"
```

Comment:
remind me what the `$viem` override is for?

Reply:
this forces all dependencies to use the version of `viem` defined in this `package.json` - this is necessary to ensure that `@renegade-fi/core` types are deemed identical between `@renegade-fi/react` and `@renegade-fi/internal-sdk`